frontend/components/transaction/BidBlock.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <TransactionMessage
+            :is-explanations-shown="isExplanationsShown"
+            :is-finished="state === 'executed'"
+            :transaction-address="auctionTransaction.transactionAddress"
```

Comment:
If `transactionAddress` exists that means it's `isFinished`, therefore, you can remove this extra prop